### PR TITLE
Roaster orders

### PR DESCRIPTION
**Pull request by lcollin**

### CODE REVIEW

File: src/components/dashboard/orders/OrderContainer.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        this.props.dispatch(getRoasterOrders(this.props.roaster.id, 0, 100)).then(() => {
+            this.setState({ordersReceived: true});
+        });
+        // this.props.dispatch(getOrders(this.props.roaster.id, 0, 100)).then(() => {
+        //     this.setState({ordersReceived: true});
+        // });
+    }
+
+    handleSuccess() {
+        if (this.state.send) {
```
Comment (ghmeier) [inline src/components/dashboard/orders/OrderContainer.js]: we don't need all of these. orders aren't created by the roaster, and the only thing they should update id getting the tracking url.

File: src/components/dashboard/orders/OrderContainer.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            this.eisActive.value = false;
+            this.edescription.value = '';
+            this.setState({selected: '', esend: false, estatus: {}, eimage: '', etags: []});
+        }
+
+        this.props.dispatch(getRoasterOrders(this.props.roaster.id, 0, 100));
+        //this.props.dispatch(getOrders(this.props.roaster.id, 0, 100));
+    }
+
+    handleEditBeans(e) {
```
Comment (ghmeier) [inline src/components/dashboard/orders/OrderContainer.js]: don't need this

File: src/components/dashboard/orders/OrderContainer.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+            if (!this.state.selected) {
+                dispatch(ActionUtil.error(400, 'Unable to upload image.'));
+                return;
+            }
+
+            dispatch(uploadImage(this.ephoto, this.state.selected));
+        });
+    }
+
+    getAddType(key) {
```
Comment (ghmeier) [inline src/components/dashboard/orders/OrderContainer.js]: don't need this.

File: src/components/dashboard/orders/OrderContainer.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+        return (status => {
+            let obj = {};
+            obj[key] = status;
+            this.setState(obj);
+        });
+    }
+
+    handleEditPhoto(file) {
```
Comment (ghmeier) [inline src/components/dashboard/orders/OrderContainer.js]: don't need this.

File: src/components/dashboard/orders/OrderContainer.js [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+    }
+
+    handleRowClick(i) {
+        const id = this.props.ids[i];
+        this.setState({selected: id});
+        let status = this.props.items[id].Status;
+        this.setState({estatus: {value: status, label: status}});
+    }
+
+    getNumber(s) {
```
Comment (ghmeier) [inline src/components/dashboard/orders/OrderContainer.js]: use this for validating packaging info.

File: src/components/dashboard/orders/OrderEdit.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
+import React, {Component, PropTypes} from 'react';
+import Select from 'react-select';
+import FileSelector from '../../FileSelector';
+import SuccessMessage from '../../SuccessMessage';
+import BeanItemImage from '../browse/BeanItemImage';
+
+class OrderEdit extends Component {
```
Comment (ghmeier) [inline src/components/dashboard/orders/OrderEdit.js]: I don't think we need this component at all. We should have a 'PackagingInfoContainer' and 'PackagingInfo' component instead.

File: src/components/dashboard/orders/Order.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+import OrderEdit from './OrderEdit';
+import BooleanFormatter from './BooleanFormatter';
+import ArrayFormatter from './ArrayFormatter';
+
+class Order extends Component {
+    constructor(props) {
+        super(props);
+
+       this._columns = [{
+            key: 'id',
```
Comment (ghmeier) [inline src/components/dashboard/orders/Order.js]: we don't need `orderId` this value doesn't make sense to users. We need the name of the item that the order references, the amount, and the date the order was created

**Review by ghmeier — CHANGES_REQUESTED** [review]

File: src/components/dashboard/inventory/InventoryContainer.js [hunk 8]
```diff
@@ -96,6 +96,7 @@ class InventoryContainer extends Component {
         }
 
         this.props.dispatch(getRoasterItems(this.props.roaster.id, 0, 100));
+        console.log("successful inventories 2");
```
Comment (jonnykry) [inline src/components/dashboard/inventory/InventoryContainer.js]: remove clg

File: src/components/dashboard/orders/OrderContainer.js [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+            isDecaf: this._ref('eisDecaf'),
+            isActive: this._ref('eisActive'),
+            description: this._ref('edescription'),
+            photo: this.handleEditPhoto.bind(this),
+            onAddBeans: this.handleEditBeans.bind(this)
+        };
+    }
+
+    componentDidMount() {
+        console.log(this.props);
```
Comment (jonnykry) [inline src/components/dashboard/orders/OrderContainer.js]: remove clg

**Review by jonnykry — APPROVED** [review]

**Review by ghmeier — APPROVED** [review]

**Pull request merged**